addons/api/addon/serializers/credential-store.js

```diff
@@ -6,7 +6,17 @@ export default class CredentialStoreSerializer extends ApplicationSerializer {
    * @method serialize
    * @param {Snapshot} snapshot
    */
-  serialize() {
+  serialize(snapshot) {
+    switch (snapshot.record.type) {
+      case 'static':
```

Comment:
Maybe `static` should be the default here?

Reply:
Suggestion: (not blocking)
If we have just 2 cases, I would say best to use an `if` for `vault` type and just return static as default. I tent to embrace `switch` when there are more than 2 cases.

Reply:
@randallmorey thanks for the catch. Updated!